packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -0,0 +1,69 @@
+import { Link } from 'react-router-dom';
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import { MAX_NUM_SUGGESTIONS } from './data/constants';
+
+const SearchSuggestions = ({ autoCompleteHits, enterpriseSlug, handleViewAllClick }) => {
+  const getLinkToCourse = (course) => `/${enterpriseSlug}/course/${course.key}`;
+  const getLinkToProgram = (program) => `/${enterpriseSlug}/program/${program.aggregation_key.split(':').pop()}`;
+
+  return (
+    <div className="suggestions" data-testid="suggestions">
+      <div>
+        <div className="mb-2 ml-2 mt-1 font-weight-bold suggestions-section">
+          Courses
+        </div>
+        {
+                    autoCompleteHits.filter(hit => hit.content_type === 'course')
+                      .slice(0, MAX_NUM_SUGGESTIONS)
+                      .map((hit) => (
+                        <Link to={getLinkToCourse(hit)} key={hit.title} className="suggestion-item" style={{ whiteSpace: 'pre-wrap' }}>
+                          <div style={{ display: 'flex' }}>
+                            {/* eslint-disable-next-line no-underscore-dangle, react/no-danger */}
+                            <div dangerouslySetInnerHTML={{ __html: hit._highlightResult.title.value }} />
```

Comment:
Is it ok to do it this way? As it can open up a cross-site scripting (XSS) attack.

Reply:
this is sent by algolia as string containing html tag to distinguish the part matching search result, shouldn't it be safe?

Reply:
Did algolia convert the text to an HTML tag? Also, please check out this link.
https://reactjs.org/docs/dom-elements.html#dangerouslysetinnerhtml